### PR TITLE
Worksheet-Module

### PR DESCRIPTION
**Pull request by vinay-pansofttech**

### CODE REVIEW

File: src/app/features/worksheet/components/worksheet-approval/worksheet-approval.component.scss [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+  align-items: center;
+  justify-content: center;
+  padding: 0.25rem;
+  margin-left: 0.5rem;
+  transition: all 0.2s ease-out;
+  color: white;
+  border-bottom: 3px solid transparent;
+}
+// .worksheet-approval-stepper-container:hover {
+//   background-color: #f8f8f8c7;
```
Comment (bharatgolagana1) [inline src/app/features/worksheet/components/worksheet-approval/worksheet-approval.component.scss:81]: remove commented code

**Review by bharatgolagana1 — APPROVED** [review]

File: src/app/features/enquiry-details/components/enquiry-details/enquiry-details.component.ts [hunk 2]
```diff
@@ -137,18 +137,31 @@ export class EnquiryDetailsComponent implements OnInit {
...
+          // if (data) {
+          //   const notificationMessage = data.outPut;
+          //   const notificationType = data.outPut.startsWith('Success') ? 'success' : 'error';
+          //   this.notificationService.showNotification(
+          //     notificationMessage,
+          //     notificationType,
+          //     'center',
+          //     'bottom'
+          //   );
+          // }
```
[the remark this answers is not in the document]
Comment (vinay-pansofttech) [reply src/app/features/enquiry-details/components/enquiry-details/enquiry-details.component.ts:151]: Done

File: src/app/features/enquiry-details/components/enquiry-details-update/enquiry-details-update.component.ts [hunk 3]
```diff
@@ -26,6 +26,7 @@ export class EnquiryDetailsUpdateComponent {
   enqId!: string;
   modeOfCommunicationValue = false;
   modeOfCommunicationControl!: FormControl;
+  docSrcTypeAttachment: number = 22;
```
[the remark this answers is not in the document]
Comment (vinay-pansofttech) [reply src/app/features/enquiry-details/components/enquiry-details-update/enquiry-details-update.component.ts:29]: Its for time being and would be handled at the API level later

File: src/app/features/enquiry-details/enquiry-details.service.ts [hunk 4]
```diff
@@ -67,28 +70,58 @@ export class EnquiryDetailsService {
...
+    } else {
+      formData.enquiryDescription.attachment = '';
+    }  
+
+    const body = new FormData();
+    body.append('soldToLEID', this.leID as string);
+    body.append('soldToContactID', formData.contactDteails.soldToContact);
+    body.append('soldToLESiteID', formData.contactDteails.soldToSite);
+    body.append('regionID', this.regionId as string);
+    body.append('salesChannelID', formData.enquiryDetailsForms.salesChannel);
```
[the remark this answers is not in the document]
Comment (vinay-pansofttech) [reply src/app/features/enquiry-details/enquiry-details.service.ts:108]: Sure Bharat, has to be handled in many places and would be done before the next PR

**Pull request merged**